files/shaders/objects_fragment.glsl

```diff
@@ -51,6 +51,10 @@ uniform mat2 bumpMapMatrix;
 
 uniform bool simpleWater;
 
+#if @grassAnimation
+uniform bool isGrass;
```

Comment:
This uniform used later doesn't exist if grass animation isn't enabled which causes issues.